document_page_quality_manual/readme/CONFIGURE.md

```diff
@@ -0,0 +1,4 @@
+To enter your work instruction template, you need to:
+
+- go to Management Systems \> Documentation \> Work Instructions
```

Comment:
i find it misleading/too general, as per usage.md we need to tick this option ![image](https://github.com/user-attachments/assets/4ee29770-b6f8-4492-ae12-6e6bf42aaa70)